webservices/common/models/itemized.py

```diff
@@ -9,25 +9,29 @@
 class BaseItemized(db.Model):
     __abstract__ = True
 
+    """These weren't exposed in the API previously, although they were in the old tables
+    gonna leave uncommented until this is clarified
+
+    candidate_office = db.Column('cand_office', db.String)
+    candidate_office_description = db.Column('cand_office_desc', db.String)
+    candidate_office_district = db.Column('cand_office_district', db.String)
+    """
     committee_id = db.Column('cmte_id', db.String, doc=docs.COMMITTEE_ID)
     committee = utils.related_committee_history('committee_id', cycle_label='report_year')
     report_year = db.Column('rpt_yr', db.Integer, doc=docs.REPORT_YEAR)
     report_type = db.Column('rpt_tp', db.String, doc=docs.REPORT_TYPE)
-    form_type = db.Column('form_tp', db.String, doc=docs.FORM_TYPE)
     entity_type = db.Column('entity_tp', db.String)
+    entity_type_desc = db.column('entity_tp_desc', db.String)
+    election_type = db.Column('election_tp', db.String, doc=docs.ELECTION_TYPE)
+    election_type_full = db.Column('election_tp_desc', db.String, doc=docs.ELECTION_TYPE)
+
+    fec_election_type_desc = db.column('fec_election_tp_desc', db.String)
+    fec_election_year = db.column('fec_election_yr', db.String)
     image_number = db.Column('image_num', db.String, doc=docs.IMAGE_NUMBER)
-    memo_code = db.Column('memo_cd', db.String)
-    memo_text = db.Column(db.String)
-    filing_type = db.Column(db.String)
     filing_form = db.Column(db.String)
     link_id = db.Column(db.Integer)
-    sub_id = db.Column(db.Integer)
-    original_sub_id = db.Column('orig_sub_id', db.Integer)
-    amendment_indicator = db.Column('amndt_ind', db.String)
```

Comment:
Yes think I meant to change that. I'll assign 'amendment_indicator' to 'action_cd'